<launch>
    <!-- node -->
    <node pkg="path_plan" type="SavePath.py" name="SavePath" output="screen">
        <param name="db_name" type="string" value="/path3.db"/>
    </node>
    <node pkg="path_plan" type="create_path.py" name="create_global_path" output="screen"/>
</launch>
